src/vm/threadsuspend.cpp

```diff
@@ -3204,7 +3206,7 @@ void __stdcall Thread::RedirectedHandledJITCase(RedirectReason reason)
 
     // We must preserve this in case we've interrupted an IL pinvoke stub before it
     // was able to save the error.
-    DWORD dwLastError = GetLastError();
+    DWORD dwLastError = GetLastError(); // BEGIN_PRESERVE_LAST_ERROR
```

Comment:
Why not macros here?

Reply:
This method has EH that the macro does not work well with.